cmake/SelectPcsx2Plugins.cmake

```diff
@@ -267,12 +267,13 @@ endif()
 #			spu2-x
 #---------------------------------------
 # requires: -SoundTouch
-#			-ALSA
-#           -Portaudio
+#           -ALSA
 #           -SDL
 #           -common_libs
+#
+# optional: -Portaudio
 #---------------------------------------
-if((PORTAUDIO_FOUND AND SOUNDTOUCH_FOUND AND SDLn_FOUND AND common_libs)
+if((SOUNDTOUCH_FOUND AND SDLn_FOUND AND common_libs)
```

Comment:
I think it's better to skip the build if PortAudio isn't found and PortAudio isn't explicitly disabled.

Reply:
Honestly, I think I'd prefer the build succeeds if it's able to succeed without portaudio. SDL's been set to the default anyways at this point on Linux, as it tends to behave better than portaudio.

Reply:
I would also prefer to let the cmake check dynamically enable/disable portaudio support and leave `-DPORTAUDIO_API=OFF` only for skipping it when portaudio is installed.